core/src/test/java/org/apache/iceberg/TestSnapshotJson.java

```diff
@@ -140,7 +140,14 @@ public void testJsonConversionWithManifestList() throws IOException {
     manifestList.deleteOnExit();
 
     try (ManifestListWriter writer =
-        ManifestLists.write(1, Files.localOutput(manifestList), id, parentId, 0)) {
+        ManifestLists.write(
+            1,
+            Files.localOutput(manifestList),
+            id,
+            parentId,
+            0,
+            /* compressionCodec */ null,
+            /* compressionLevel */ null)) {
```

Comment:
Probably not worth updating this, can you revert it?

Reply:
Done.